_integration-test/run.sh

```diff
@@ -130,6 +130,42 @@ cat debug.log
 echo '------------------------------------------'
 echo "${_endgroup}"
 
+echo "${_group}Test symbolicator works ..."
+SENTRY_ORG="${SENTRY_ORG:-sentry}"
+SENTRY_PROJECT="${SENTRY_PROJECT:-native}"
+SENTRY_TEAM="${SENTRY_TEAM:-sentry}"
+# First set up a new project if it doesn't exist already
+PROJECT_JSON=$(jq -n -c --arg name "$SENTRY_PROJECT" --arg slug "$SENTRY_PROJECT" '$ARGS.named')
+NATIVE_PROJECT_ID=$(sentry_api_request "api/0/teams/$SENTRY_ORG/$SENTRY_TEAM/projects/" | jq -r '.[]|select(.slug == "'"$SENTRY_PROJECT"'")|.id')
```

Comment:
nit: `"'"$SENTRY_PROJECT"'" -> "'$SENTRY_PROJECT'"`

Reply:
That wouldn't escape the variable. Because its the same as `'string' + $var + 'end'`, whereas we want `'string' + "$var" + 'end'`. Not really an issue since this is only a single word, but for consistency's sake I try to always quote variables.